docs/index.md

```diff
@@ -76,11 +76,11 @@ For the sake of data protection, we do not return detailed messages for most of
 ### Rich-text formatting
 
 Resources which include `RichText` (e.g. `Item` with its `description` field) can be optionally handled by the server in Markdown format:
-- to send a request payload which includes Markdown-formatted `RichText`, change `Content-Type` header to `application/json+markdown`
-- to receive a response payload which includes `RichText` formatted as Markdown, change `Accept` header to `application/json+markdown`
+- to send a request payload which includes Markdown-formatted `RichText`, change `Content-Type` header to `application/vnd.airfocus.markdown+json`
+- to receive a response payload which includes `RichText` formatted as Markdown, change `Accept` header to `application/vnd.airfocus.markdown+json`
```

Comment:
Should we also explain the format of the "markdown RichText" object? It's not a drop-in replacement for the old header since it now has to have the `{ "isRichText": true, "markdown": "..."}` structure instead of just `"..."`.